### DISCUSSION TITLE
Spine 导入到editor后与设计稿不一样

### DISCUSSION THREAD

**Discussion by JXD28** [galacean/discussion#117] [Editor]
![image](https://img.alicdn.com/imgextra/i2/O1CN01M16wvo1hy3oGGRRDG_!!6000000004345-0-tps-1370-694.jpg)
Spine 版本是3.8 

素材:
"https://gw.alicdn.com/imgextra/i2/O1CN014zvnCQ1Yqt416NMYQ_!!6000000003111-2-tps-1491-622.png",
 "https://g.alicdn.com/eva-assets/097c0a76532bab33724b9e6c308807a4/0.0.1/tmp/78b308e/9e57381c-cdd2-4e7f-8bc0-84af6e5269a9.json",
"https://g.alicdn.com/eva-assets/c6f516d4d78488a5f0e2e9a9e1ac18bd/0.0.1/tmp/ddbdff7/2adeb8b8-2838-48f0-b116-f8fbefc9c6cb.atlas"

**Comment by singlecoder** [comment]
有在 Spine 软件里面的效果么
